addons/web/static/src/legacy/js/views/calendar/calendar_model.js

```diff
@@ -115,13 +116,34 @@ return AbstractModel.extend({
         if (filter.value === 'all') {
             Filter.all = filter.active;
         }
+        if (filter.value === 'check_all') {
+            // Select all numeric filters and update the checkboxes
+            this.filter_check_all[filter.fieldName] = filter.active;
+            // Deactivate the All selection
+            Filter.all = false;
+            if (filter.active) {
+                // Check every displayed filters except the 'all' filter
+                Filter.filters.filter(el => el.value !== 'all').forEach(element => {
+                    element.active = true;
+                });
```

Comment:
```suggestion
                Filter.filters.filter(el => el.value !== 'all').forEach(element => element.active = true;);
```

Reply:
Also, it is possible to filter the values different from `all` inside the forEach, therefore looping through the filters only once.